Mathlib/Analysis/CStarAlgebra/Hom.lean

```diff
@@ -0,0 +1,79 @@
+/-
+Copyright (c) 2024 Jireh Loreaux. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jireh Loreaux
+-/
+
+import Mathlib.Analysis.CStarAlgebra.ContinuousFunctionalCalculus.Order
+
+/-! # Properties of C⋆-algebra homomorphisms
+
+Here we collect properties of C⋆-algebra homomorphisms.
+
+## Main declarations
+
++ `NonUnitalStarAlgHom.norm_map`: A non-unital star algebra monomorphism of complex C⋆-algebras
+  is isometric.
+-/
+
+namespace NonUnitalStarAlgHom
+
+variable {F A B : Type*}
+variable [NonUnitalNormedRing A] [CompleteSpace A] [StarRing A] [CStarRing A]
+variable [NormedSpace ℂ A] [IsScalarTower ℂ A A] [SMulCommClass ℂ A A] [StarModule ℂ A]
+variable [NonUnitalNormedRing B] [CompleteSpace B] [StarRing B] [CStarRing B]
+variable [NormedSpace ℂ B] [IsScalarTower ℂ B B] [SMulCommClass ℂ B B] [StarModule ℂ B]
+variable [FunLike F A B] [NonUnitalAlgHomClass F ℂ A B] [NonUnitalStarAlgHomClass F ℂ A B]
+
+open CStarAlgebra Unitization in
+/-- A non-unital star algebra monomorphism of complex C⋆-algebras is isometric. -/
+lemma norm_map (φ : F) (hφ : Function.Injective φ) (a : A) : ‖φ a‖ = ‖a‖ := by
+  /- by passing to the unitization is functorial, and it is an isometric embedding, we may assume
+  that `φ` is a unital star algebra monomorphism and that `A` and `B` are unital C⋆-algebras. -/
```

Comment:
```suggestion
  /- Since passing to the unitization is functorial, and it is an isometric embedding, we may assume
  that `φ` is a unital star algebra monomorphism and that `A` and `B` are unital C⋆-algebras. -/
```